### ISSUE TITLE
Tag translate #89 

### ISSUE THREAD

**Issue by Vynits** [simplicialsoftware/nebulous-translations#526]
Hi Grog.. I just want to ask, how long do I have to wait to get the Tag translate.. 🙏
![Screenshot_2024-09-01-22-36-13-94](https://github.com/user-attachments/assets/c687ce5f-c42d-4e4d-b03e-c2e00f9a5f3a)
![Screenshot_2024-09-01-22-35-46-15](https://github.com/user-attachments/assets/3d9cbc36-ad7d-45c9-8b69-47b56914862e)

**Comment by JSI31** [comment]
Did you make the correct changes?

**Comment by Vynits** [comment]
It's all correct, Greg asked me to send the ID, I have sent the ID to the email.

**Comment by JSI31** [comment]
Did he tell you it was true? I agree And send you something like this @Jeii20 agreed to this pull request.

**Comment by Vynits** [comment]
If the proof is still not enough, I can send it via Discord..

**Comment by JSI31** [comment]
Yes, if he does not send you such a message and you must review and make changes until they agree

**Comment by Vynits** [comment]
What should I do again, I have created a ticket on discord but no one has replied, I am confused about what to do to get the translated tag

**Comment by JSI31** [comment]
My dear, you must follow the correct steps. It depends on your cleverness and effort. Focus on what are the incorrect things in the translation and correct them.

**Comment by Vynits** [comment]
![Screenshot_2024-09-02-00-24-21-11](https://github.com/user-attachments/assets/ace3350b-c314-4cae-a88f-407f0977cb1f)
I have made a lot

**Comment by JSI31** [comment]
I'm so sorry. This is the rules for getting your mark

**Comment by Vynits** [comment]
Do I have to make it again?

**Comment by Vynits** [comment]
![Screenshot_2024-09-02-00-29-16-59](https://github.com/user-attachments/assets/b8da22fb-8693-498a-a60a-fa50289b55f9)
Why did Greg ask me to send the ID, if it won't be processed?

**Comment by JSI31** [comment]
When did I send it to you? If you are asked to send your ID, they will agree

**Comment by Vynits** [comment]
There is already proof, Grog asked me to send the ID and I have sent my ID...

**Comment by JSI31** [comment]
Well just wait what he tells you

**Comment by Vynits** [comment]
Okay, sorry to bother you.

**Comment by JSI31** [comment]
hopefully

**Comment by ZheiNB** [comment]
Interesting.

**Comment by JSI31** [comment]
You're welcome

**Comment by AlExX2457** [comment]
How long did i have to wait to can take Tag Translator from Romanian, Many words are translated badly, I translated them well! My id is 20713867 @grog444 @JSI31 , Some words are not translated correctly honestly, others were approved by that Nebulos, mine never, why isn't justice done!!!! @grog444 I want explanations!

**Comment by AlExX2457** [comment]
@JSI31 @grog444 @indra12356 I want justice, and answers, that's why I'm wrong in the Romanian translation, I translate well, why am I not getting justice!!!

**Comment by Vynits** [comment]
> Berapa lama saya harus menunggu untuk dapat menggunakan Tag Translator dari bahasa Rumania? Banyak kata yang diterjemahkan dengan buruk, tetapi saya menerjemahkannya dengan baik! ID saya adalah 20713867@grog444 @JSI31, Beberapa kata tidak diterjemahkan dengan benar sejujurnya, yang lain disetujui oleh Nebulos itu, milikku tidak pernah, mengapa keadilan tidak ditegakkan!!!!@grog444Saya ingin penjelasan!

Please make your own report, don't follow other people's reports..

**Comment by Vynits** [comment]
@JSI31 This is how it continues @grog444 Please help

**Comment by JSI31** [comment]
Watch the tutorial, my friend

**Comment by Vynits** [comment]
Who's video tutorial @Jeii20 I have watched it

**Comment by Vynits** [comment]
![Screenshot_2024-09-02-16-20-29-35](https://github.com/user-attachments/assets/44a0a498-7e29-4d75-88fb-7f9938a02d14)
@JSI31 Can you resend the file to me.. I will create a new one..

**Comment by Jeii20** [comment]
Hi, @Vynits, @JSI31 and @AlExX2457,

Please avoid the **Issues** tab to discuss this topic. This can only be used to report untranslated texts or similar. Also, please stop marking the developer several times, this is disturbing. I will mark this issue as completed, as there is an invalid subject. Thanks for understanding.
